panic when delete non-existent vips from load balancer in mutation operation

1. create a lb with vips
2. delete non-existent vips from load balancer in mutation operation
3. panic
```
panic: reflect: call of reflect.Value.Interface on zero Value

goroutine 67 [running]:
reflect.valueInterface({0x0?, 0x0?, 0x17129e0?}, 0xc0?)
	/usr/local/go/src/reflect/value.go:1435 +0x10e
reflect.Value.Interface(...)
	/usr/local/go/src/reflect/value.go:1430
github.com/ovn-org/libovsdb/server.mutateDelete({0x13618e0?, 0xc00044a9f0?}, {0x13618e0, 0xc00044aa80?})
	/lee/go/pkg/mod/github.com/ovn-org/libovsdb@v0.0.0-20220603173653-bbc32842d174/server/mutate.go:167 +0x7eb
github.com/ovn-org/libovsdb/server.mutate({0x13618e0?, 0xc00044a9f0?}, {0xc000179e24?, 0x13618e0?}, {0x13618e0?, 0xc00044aa80?})
	/lee/go/pkg/mod/github.com/ovn-org/libovsdb@v0.0.0-20220603173653-bbc32842d174/server/mutate.go:38 +0x2a8
github.com/ovn-org/libovsdb/server.(*Transaction).Mutate(0xc0000bb1a8, {0xc0003e6cf0?, 0x7c4e0c7028360a05?}, {0xc000179e00, 0xd}, {0xc0000de840, 0x1, 0x4}, {0xc0000de780, 0x1, ...})
	/lee/go/pkg/mod/github.com/ovn-org/libovsdb@v0.0.0-20220603173653-bbc32842d174/server/transact.go:422 +0xdf4
github.com/ovn-org/libovsdb/server.(*OvsdbServer).transact(0xc00022a370, {0xc000179dd0, 0xe}, {0xc0004a9790, 0x1, 0xc000199398?})
	/lee/go/pkg/mod/github.com/ovn-org/libovsdb@v0.0.0-20220603173653-bbc32842d174/server/transact.go:58 +0x107e
github.com/ovn-org/libovsdb/server.(*OvsdbServer).Transact(0xc00022a370, 0xc000488000?, {0xc0003f6b40, 0x2, 0xc000199808?}, 0xc00053d020)
	/lee/go/pkg/mod/github.com/ovn-org/libovsdb@v0.0.0-20220603173653-bbc32842d174/server/server.go:238 +0x672
```

--- a/server/mutate.go
+++ b/server/mutate.go
@@ -164,7 +164,7 @@ func mutateDelete(current, value interface{}) (interface{}, interface{}) {
 			vvk := iter.Key()
 			vvv := iter.Value()
 			vcv := vc.MapIndex(vvk)
-			if reflect.DeepEqual(vcv.Interface(), vvv.Interface()) {
+			if vcv.IsValid() && reflect.DeepEqual(vcv.Interface(), vvv.Interface()) {
 				diff.SetMapIndex(vvk, vcv)
 				vc.SetMapIndex(vvk, reflect.Value{})
 			}

--- a/server/mutate_test.go
+++ b/server/mutate_test.go
@@ -376,6 +376,22 @@ func TestMutateDelete(t *testing.T) {
 			},
 		},
 		{
+			"delete non-existent key value pairs",
+			map[string]string{
+				"foo": "bar",
+				"baz": "quux",
+			},
+			ovsdb.MutateOperationDelete,
+			map[string]string{
+				"key": "value",
+			},
+			map[string]string{
+				"foo": "bar",
+				"baz": "quux",
+			},
+			nil,
+		},
+		{
 			"delete keys",
 			map[string]string{
 				"foo": "bar",

